air/src/trace/snapshots/miden_air__trace__tests__all_chiplet_column_ranges.snap

```diff
@@ -0,0 +1,34 @@
+---
+source: air/src/trace/tests.rs
+expression: layout
+---
+ALL CHIPLET COLUMN RANGES
+================================
+
+Chiplet Selector Counts:
+  - Hasher selectors: 1
+  - Bitwise selectors: 2
+  - Memory selectors: 3
+  - ACE selectors: 4
+  - Kernel ROM selectors: 5
+
+Hasher Chiplet:
+  Trace offset: 52
+  Selector range: 52..55
+  State range: 55..67
+  Capacity range: 55..59
+  Rate range: 59..67
+  Node index: 67
+
+Bitwise Chiplet:
+  Trace offset: 53
+  Selector index: 53
+  Input A range: 56..60
```

Comment:
Why are we reporting this here as well as in the Bitwise trace snapshot? We should either put all the info here, or in the individual chiplet snapshots. The same applies to the other chiplets.